bigchaindb/backend/query.py

```diff
@@ -33,6 +33,20 @@ def store_asset(connection, asset):
     raise NotImplementedError
 
 
+@singledispatch
+def store_assets(connection, assets):
+    """Write an asset to the asset table.
+
+    Args:
+        assets ([dict]): list of assets.
```

Comment:
Is the type a dict or a list of dicts?

The docstring for `store_metadata` (or whatever it gets renamed as) should also say that it stores a list.